README.md

```diff
@@ -389,30 +391,72 @@ mkdir gx_logs
+    <env id="GALAXY_LIB">/cluster_storage/galaxy/galaxy_export/galaxy-central/lib</env>
+    <env id="PYTHONPATH">/cluster_storage/galaxy/galaxy_export/galaxy-central/lib</env>
+    <param id="embed_metadata_in_job">True</param>
+  </destination>
+```
+In this way, python tools on the cluster are able to use the galaxy libraries.
 
-These files from the cluster must be copied to the `/export` mount point (i.e., `/data/galaxy` on the host if using below command) accessible to Galaxy before starting the container. This must be done regardless of which Slurm daemons are running within Docker. At start, symbolic links will be created to these files to `/etc` within the container, allowing the various Slurm functions to communicate properly with your cluster. In such cases, there's no reason to run `slurmctld`, the Slurm controller daemon, from within Docker, so specify `-e "NONUSE=slurmctld"`. Unless you would like to also use Slurm (rather than the local job runner) to run jobs within the Docker container, then alternatively specify `-e "NONUSE=slurmctld,slurmd"`.
+More information can be found [here](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/admin/framework_dependencies.rst#managing-dependencies-manually)
+and
+[here](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/admin/framework_dependencies.rst#galaxy-job-handlers).
 
-Importantly, Slurm relies on a shared filesystem between the Docker container and the execution nodes. To allow things to function correctly, each of the execution nodes will need `/export` and `/galaxy-central` directories to point to the appropriate places. Suppose you ran the following command to start the Docker image:
+### Using an external Slurm cluster <a name="Using-an-external-Slurm-cluster" /> [[toc]](#toc)
 
-```sh
-docker run -d \
-    -e "NONUSE=slurmd,slurmctld" \
-    -p 80:80 \
-    -v /data/galaxy:/export \
-    bgruening/galaxy-stable
-```
+It is often convenient to configure Galaxy to use a high-performance cluster for running jobs. To do so, two files are required:
+
+1. munge.key
+2. slurm.conf
 
-You would then need the following symbolic links on each of the nodes:
+These files from the cluster must be copied to the `/export` mount point (i.e., `/cluster_storage/galaxy/galaxy_export/` on the host if using below command) accessible to Galaxy before starting the container. This must be done regardless of which Slurm daemons are running within Docker. At start, symbolic links will be created to these files to `/etc` within the container, allowing the various Slurm functions to communicate properly with your cluster. In such cases, there's no reason to run `slurmctld`, the Slurm controller daemon, from within Docker, so specify `-e "NONUSE=slurmctld"`. Unless you would like to also use Slurm (rather than the local job runner) to run jobs within the Docker container, then alternatively specify `-e "NONUSE=slurmctld,slurmd"`.
```

Comment:
Can we also highlight the fact that this is only needed if you can not have a /export mount point on your cluster?